Cargo.toml

```diff
@@ -54,9 +54,54 @@ async-tokio = ["tokio"]
 ## UTF-16 will not work (therefore, `quick-xml` is not [standard compliant]).
 ##
 ## List of supported encodings includes all encodings supported by [`encoding_rs`]
-## crate, that satisfied the restriction above.
+## crate, that satisfied the restriction above. So, the following encodings are
+## **not supported**:
```

Comment:
There's a lot of redundancy between this, the lines above, and the lines below that can probably be cleaned up.

Reply:
I tried to remove redundancy, if you have something else in mind, please make a suggestion.